changelogs/unreleased/cbus-lcpipe.md

```diff
@@ -0,0 +1,4 @@
+## feature/build
```

Comment:
`on_flush` triggers seem unused in all the non-test code. You can delete them from `cpipe` too.

Reply:
Yes, you are right, maybe we should make this in a separate PR? Looks like this not about lcpipe, but about just cpipe refactoring

Reply:
I would simply make it in a separate commit in the same PR.